serverless/src/getAccessMethods/getServiceOptionDefinitions.js

```diff
@@ -0,0 +1,55 @@
+import request from 'request-promise'
+import jwt from 'jsonwebtoken'
+import 'array-foreach-async'
+
+import {
+  getEarthdataConfig,
+  getSecretEarthdataConfig,
+  getClientId
+} from '../../../sharedUtils/config'
+
+export const getServiceOptionDefinitions = async (serviceOptionDefinitions, jwtToken) => {
+  const forms = []
+
+  await serviceOptionDefinitions.forEachAsync(async (serviceOptionDefinition, index) => {
+    const { id: guid } = serviceOptionDefinition
+
+    const url = `${getEarthdataConfig('prod').cmrHost}/legacy-services/rest/service_option_definitions/${guid}.json`
+
+    // Get the access token and clientId to build the Echo-Token header
+    const { clientId, secret } = getSecretEarthdataConfig('prod')
+
+    const verifiedJwtToken = jwt.verify(jwtToken, secret)
```

Comment:
It looks like we do this exact bit of code to extract the token in quite a few places now. Maybe worth turning it into a util to prevent duplication.

Reply:
Appears that this was taken care of in his other PR 🤷‍♂